handbook/ce/engaging-other-teams.md

```diff
@@ -1,5 +1,169 @@
 # Engaging Other Teams
 
-This is forthcoming (being worked on by [Virginia](https://about.sourcegraph.com/company/team#virginia-ulrich-she-her)) and is meant to consolidate/revise [CE-to-Engineering issue handover process](ce_to_eng_handover.md) and [How to route questions from customers](routing_questions.md). It will also touch on the fact that we never want CSEs to be gatekeepers and have a template for details CSEs will include when seeking help from engineering. And it will include details about engaging marketing, etc.
+Our goal as Customer Support is to only need help from engineering no more than 10% of the time (not counting when we file defects). It's okay that we need help, and we will need help, and we also continually take what we learn each time we do back to the rest of the team (and our product documentation, as relevant).
 
-Once drafted, Virginia will ask that all eng managers and CEs review
\ No newline at end of file
+When we need help, we seek it consistently, following the steps outlined here. We created these steps to ensure we:
+
+* Take the work as far as we can
+* Exhaust our collective expertise
+* Minimize the time added to the customer experience whilst they wait for help
+
+At no point are we meant to be gate keepers or go-betweens. Generally, when a CE needs help, they will come to us. They may also still go directly to engineering for other questions they have. And when we engage engineering, if the CE wants to be included, be sure to do so. This is especially relevant for customers still in the pre-sales process where everything is just a bit more sensitive. 
+
+## Step 1: Gather all the details/context and set expectations with the customer
+
+The more we know, the easier it will be to find the answer. For example:
+
+* Do we have updated details about the customer's environment?
+* Do we have the steps to be able to reproduce the issue and relevant screenshots and logs?
+* Do we know if the customer made any other changes to their environment recently (this could be anything, not just related to Sourcegraph)?
+* Do we have a few examples of users who are impacted and those who are not if the issue is not impacting all of their users?
+* Do we have an understanding of the level of urgency the customer and/or our CE/Sales counterparts are feeling to resolve the issue?
+
+Never say "I need to ask engineering" or anything that can erode trust with the customer in your ability. It's okay that you need help! Instead say things like:
+
+* "I want to talk to some of my teammates about this one..."
+* Or, if it's a defect, "I've filed the issue in our public tracker. I'll let you know when it's resolved and you can follow along here...."
```

Comment:
Are we moving away from using the /customer private tracker? (I had an issue get moved out of the public tracker and into the private one because it was customer reported, is why I ask.)

Reply:
Good catch, maybe we should change the example or clarify. The intent was that the customer issue was tracked separately than a defect, even if they can be linked to each other because the customer issue itself could be resolved in many cases separately and it also used to share internal next steps, previous logs, or other relevant information, while the public one should only contain general information about the defect itself.

Reply:
This is explained elsewhere (which tracker to use), so I will just remove this example entirely so no one gets confused about which tracker to use for what.

Reply:
sgtm!